app/views/users/index.html.erb

```diff
@@ -1,2 +1,15 @@
 <h1>Users#index</h1>
```

Comment:
- [ ] I would like to recommend you remove this `<h1> ` element and make sure that your index view page match this  [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) as required for this project.  for example, the user photo is square, not rounded. Please fix it :+1: